lib/features/chat/notifiers/chat_room_notifier.dart

```diff
@@ -57,32 +62,31 @@ class ChatRoomNotifier extends StateNotifier<ChatRoom> {
         return;
       }
 
-      final eventStore = ref.read(eventStorageProvider);
 
-      if (!await eventStore.hasItem(event.id!)) {
-        await eventStore.putItem(
-          event.id!,
-          {
-            'id': event.id,
-            'created_at': event.createdAt!.millisecondsSinceEpoch ~/ 1000,
-          },
-        );
-      }
```

Comment:
The Chat events have to be saved to the EventStore so the App knows it has downloaded the event and doesn't constantly notify the user that they have new messages